shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -511,7 +528,10 @@ private void resetWillNotDraw(boolean isAccessibilityEnabled, boolean isTouchExp
    * See {@link #detachFromFlutterEngine()} for information on how to detach from a
    * {@link FlutterEngine}.
    */
-  public void attachToFlutterEngine(@NonNull FlutterEngine flutterEngine) {
+  public void attachToFlutterEngine(
+      @NonNull FlutterEngine flutterEngine,
+      @NonNull PlatformViewsController platformViewsController
```

Comment:
So now if I am just integrating via a FlutterView, I have to create a PlatformViewController too?

Reply:
Yeah, with the way `PlatformViewsController` is integrated with the embedding right now, it's a requirement. If we'd like to make it optional then we'd need to make that change within the platform views subsystem, itself, because right now I'm just matching how platform views integrates with the old embedding.